src/pages/schedule.re

```diff
@@ -122,7 +122,7 @@ let make = _children => {
         <time dateTime="2018-05-11">
           ("11" |> s)
           <sup> ("th" |> s) </sup>
-          (" May, Friday" |> s)
+          (" May, Friday at ImpactHub" |> s)
```

Comment:
Just thought that it would be great to link to "For Attendees" page from the location.